src/app/components/MediaPlayer/utils/buildSetting.test.ts

```diff
@@ -0,0 +1,82 @@
+import { MediaBlock } from '../types.d';
+import buildConfig from './buildSettings';
+import blocks from '../fixture';
+
+describe('buildSettings', () => {
+  it('Should process an AresMedia block into a valid playlist item.', () => {
+    const result = buildConfig({
+      id: 'testID',
+      blocks,
+      pageType: 'article',
+      counterName: null,
+    });
+    expect(result).toStrictEqual({
+      product: 'news',
+      superResponsive: true,
+      playlistObject: {
+        title: 'Five things ants can teach us about management',
+        holdingImageURL:
+          'https://ichef.test.bbci.co.uk/images/ic/512xn/p01k6mtv.jpg',
+        items: [{ duration: 191, kind: 'programme', versionID: 'p01k6msp' }],
+        guidance: 'Contains strong language and adult humour.',
+      },
+    });
+  });
+
+  it('Should include the mediator parameter if we are on a test url.', () => {
+    process.env.NODE_ENV = 'development';
```

Comment:
Should we be deleting this in an `afterEach` just in case it impacts other tests? (Given that the default for a local environment is `development`)

eg. `delete process.env.NODE_ENV`